### ISSUE TITLE
Stats parsing is broken when using "all: true" and Webpack 5

### ISSUE THREAD

**Issue by slavafomin** [webpack-contrib/webpack-bundle-analyzer#417]
Hello!

Thank you for this great tool!

#### Issue description

However, I've encountered a situation when stats file parsing is broken when using the following stats configuration:

```javascript
const statsConfig = {
  all: true,
  context: projectPath,
  source: false,
}
```

and Webpack `5.22.0`.

#### Technical info

The problem is in the [assetHasModule](https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/37f048a500ebaa2394350183dee7b5c4c09d8489/src/analyzer.js#L191) function:

```javascript
function assetHasModule(statAsset, statModule) {
  // Checking if this module is the part of asset chunks
  return statModule.chunks.some(moduleChunk => statAsset.chunks.includes(moduleChunk));
}
```

It raises the following error:

`TypeError: Cannot read property 'some' of undefined`

because `statModule.chunks` is `undefined` in this case.

In my example the module stat object has the following structure:

```json5
{
  "type": "modules by path",
  "name": "…node_modules/.pnpm/core-js@3.8.3/node_modules/core-js/internals/*.js",
  "children": [
    // A big list of modules here
  ],
  "size": 22868,
  "sizes": {
    "javascript": 22868
  }
}
```

---

Also, when I change the stats config to use "verbose" preset, the error goes away, but when I view the bundle layout in "parsed" format, it only renders the entry point module. I think something is broken on a more fundamental level.

---

[I've uploaded](https://drive.google.com/file/d/1wpWsx6ELqt0Dy5kfHnlylyOnd3QQtD6Z/view) the obfuscated version of my stats.json file generated with the latest version of the Webpack using "verbose" preset.

**Comment by valscion** [comment]
Thanks for the issue! This might be a regression stemming from https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/395 where the `assetHasModule` previously used lodash's `_.some` method, which likely works better when the value given in is `undefined`.

**Comment by valscion** [comment]
Should be fixed in v4.4.1, thanks to @deanshub via #433. Let us know if the fix works for you ☺️
